tests/unit/type/hash-field-expire.tcl

```diff
@@ -324,9 +324,9 @@ start_server {tags {"external:skip needs:debug"}} {
             r del myhash
             r hset myhash field1 value1
 
-            r hexpireat myhash [expr {[clock seconds] + 2}] NX FIELDS 1 field1
-            assert_range [r hpttl myhash FIELDS 1 field1] 1000 2000
-            assert_range [r httl myhash FIELDS 1 field1] 1 2
+            r hexpireat myhash [expr {[clock seconds] + 10}] NX FIELDS 1 field1
+            assert_range [r hpttl myhash FIELDS 1 field1] 5000 10000
+            assert_range [r httl myhash FIELDS 1 field1] 5 10
```

Comment:
I think it is sufficient to only modify the assert of hpttl to something like:
```
assert_range [r hpttl myhash FIELDS 1 field1] 950 2000
```
It more reflects the expected output and gives enough spare.

Reply:
it's still close between 950 and 999, what about relaxing a bit more?

Reply:
As the error says, `Expected '999' to be between to '1000' and '2000'`, the problem is the actual command executing time has a delay. So we should make the start number of the assertion be small enough which is less than  the `hpttl` result.

Reply:
Updated accordingly, could you take a look?